### PR TITLE
Use logger to get rid of puts

### PR DESCRIPTION
**Pull request by kenhys**
Closes: #201

### CODE REVIEW

**Review by cosmo0920 — APPROVED** [review]
LGTM.

nits: We're already on Ruby 2.5.
How about using keyword arguments instead of self parsing Hash objects?

File: td-agent/Rakefile [hunk 1]
```diff
@@ -1151,7 +1163,7 @@ end
 class CompressDMG
   include Rake::DSL
 
-  def initialize(version, window_bounds, pkg_position)
+  def initialize(version, window_bounds, pkg_position, logger)
```
Comment (cosmo0920) [inline td-agent/Rakefile:1166]: This modification causes error?

```
rake aborted!
ArgumentError: wrong number of arguments (given 3, expected 4)
/Users/runner/work/td-agent-builder/td-agent-builder/td-agent/Rakefile:1166:in `initialize'
/Users/runner/work/td-agent-builder/td-agent-builder/td-agent/Rakefile:1367:in `new'
/Users/runner/work/td-agent-builder/td-agent-builder/td-agent/Rakefile:1367:in `block in run_pkgbuild'
/Users/runner/work/td-agent-builder/td-agent-builder/td-agent/Rakefile:1345:in `run_pkgbuild'
/Users/runner/work/td-agent-builder/td-agent-builder/td-agent/Rakefile:1336:in `block (2 levels) in define'
/usr/local/lib/ruby/gems/2.6.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => dmg:selfbuild
(See full trace by running task with --trace)
rake aborted!
Command failed with status (1): [/usr/local/Cellar/ruby@2.6/2.6.6_1/bin/rub...]
/Users/runner/work/td-agent-builder/td-agent-builder/Rakefile:36:in `block (3 levels) in define_bulked_task'
/Users/runner/work/td-agent-builder/td-agent-builder/Rakefile:35:in `block (2 levels) in define_bulked_task'
/Users/runner/work/td-agent-builder/td-agent-builder/Rakefile:34:in `each'
/Users/runner/work/td-agent-builder/td-agent-builder/Rakefile:34:in `block in define_bulked_task'
/usr/local/lib/ruby/gems/2.6.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => dmg:selfbuild
(See full trace by running task with --trace)
##[error]Process completed with exit code 1.
````

Comment (cosmo0920) [reply td-agent/Rakefile:1166]: I'd fixed in https://github.com/fluent-plugins-nursery/td-agent-builder/pull/203/commits/bffe7ecefa6a5e2da1670e75dc4cb845013968cf.

Comment (kenhys) [reply td-agent/Rakefile:1166]: Ah, It should be fixed.

Comment (kenhys) [reply td-agent/Rakefile:1166]: Thanks.

**Pull request merged**